packages/uma-auth-client/src/utils/isValidUma.ts

```diff
@@ -0,0 +1,9 @@
+export const isValidUma = (address?: string): boolean => {
+  if (!address) {
+    return false;
+  }
+
+  return /^\$[a-zA-Z0-9\-_.+]+@([a-zA-Z0-9-_.+:]+\.)+[a-zA-Z0-9:]+$/.test(
```

Comment:
This will not allow non-uma lightning addresses. Is that a problem for manual connections, etc? Should this really just validate lightning address structure (or uma)?

Reply:
This is util is just for checking if a provided string/address is the uma format

We still display non-uma addresses in the UI but in a different way